config/samples/backends/lvm-iscsi/iscsid.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     systemd:
       units:
       - enabled: true
-        name: iscsid.service
\ No newline at end of file
+        name: iscsid.service
```

Comment:
what was changed here I cannot see the difference

Reply:
this change removed extra line.

Reply:
Actually, it added a newline at the end of the file. Not an extra newline, just one at the end of the existing line.